src/main/java/Archibald.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class Archibald {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
+    public static void printArchibaldResponse(String message) {
+        System.out.println("____________________________________________________________");
+        System.out.println(message);
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(String input) {
+        if (taskCount >= tasks.length) {
+            printArchibaldResponse("Error: Task list is full!");
+            return;
+        }
+
+        String[] parts = input.split(" ", 2);
```

Comment:
When splitting the input and accessing parts[0], there is a risk of ``ArrayIndexOutOfBoundsException``. Maybe could add some validations before accessing the array?